<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>782</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>581</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTabWidget" name="RightBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>231</width>
      <height>585</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="settings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QGroupBox" name="ArdSettingsBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Ардуино</string>
      </property>
      <widget class="QGroupBox" name="ArdPortBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>COM-Порт</string>
       </property>
       <widget class="QLabel" name="ArdComPort">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Fixedsys</family>
          <pointsize>25</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="ArdComPortSelect">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>8</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="ArdSpeedBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Скорость</string>
       </property>
       <widget class="QLCDNumber" name="ArdSpeed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="digitCount">
         <number>6</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>115200.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>115200</number>
        </property>
       </widget>
       <widget class="QToolButton" name="ArdSpeedSelect">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="ArdSettings">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Настройки</string>
       </property>
       <widget class="QCheckBox" name="ArdDataDispatch">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Отправка данных</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Csv">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>211</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>CSV</string>
      </property>
      <widget class="QGroupBox" name="FileNameBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Имя файла</string>
       </property>
       <widget class="QLineEdit" name="FileName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>output</string>
        </property>
        <property name="maxLength">
         <number>21</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>.csv</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="DataRecord">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>191</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Запись данных</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QCheckBox" name="HelmetRec">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Шлем</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="LhandRec">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Л. Рука</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="RhandRec">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>П. Рука</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="LkneeRec">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Л. Колено</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="RkneeRec">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>П. Колено</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="LshinRec">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Л. Голень</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="RshinRec">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>П. Голень</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="SpeedRec">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>140</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Скорость</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="StatusRec">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="StopRecordButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Стоп</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StartRecordButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SpeedBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>81</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Скорость</string>
    </property>
    <widget class="QSlider" name="SpeedSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>22</width>
       <height>141</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="SpeedBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QToolButton" name="SpeedLock">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>🔓</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AccelerationBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>81</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Ускорение</string>
    </property>
    <widget class="QSlider" name="AccelerationSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>22</width>
       <height>141</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="AccelerationBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
    <widget class="QToolButton" name="AccelerationLock">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>🔓</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CurrentSpeed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Текущая скорость</string>
    </property>
    <widget class="QLCDNumber" name="Display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>500</width>
       <height>100</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Dec</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Console">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>501</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Консоль</string>
    </property>
    <widget class="QTextBrowser" name="ConsoleOutput">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ControlBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление</string>
    </property>
    <widget class="QPushButton" name="StartButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TrainingBar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Обучение</string>
    </property>
    <widget class="QPushButton" name="MoveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Движение</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StayButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Остановка</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ArduinoBar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Ардуино</string>
    </property>
    <widget class="QGroupBox" name="ardControl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Управление</string>
     </property>
     <widget class="QPushButton" name="Connect">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Подключить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Disconnect">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Отключить</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="StatusBar">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Состояние </string>
     </property>
     <widget class="QLabel" name="Status">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0004;&quot;&gt;Отключено&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
